roles/importer/files/importer/checkpointR8x/cp_const.py

```diff
@@ -17,24 +17,39 @@
 original_obj_uid = "85c0f50f-6d8a-4528-88ab-5fb11d8fe16c"
 # used for nat only (both svc and nw obj)
 
-nw_obj_table_names = [
+local_nw_obj_table_names = [
     'hosts', 'networks', 'groups', 'address-ranges', 'multicast-address-ranges', 'groups-with-exclusion',
     'gateways-and-servers', 'simple-gateways',
-    'dns-domains', 'updatable-objects-repository-content',
+    'dns-domains', 
     'interoperable-devices', 'security-zones', 
-    'access-roles', 'updatable-objects', 'dynamic-objects',
+    'access-roles',
     'CpmiVoipSipDomain', 'CpmiVoipMgcpDomain', 'gsn_handover_group'
 ]
 
+# the global objects need to be fetched only once per super manager
+global_nw_obj_table_names = [ 'updatable-objects-repository-content', 'updatable-objects', 'dynamic-objects' ]
+
+nw_obj_table_names = local_nw_obj_table_names + global_nw_obj_table_names
+
 # simple as in: no groups
 simple_svc_obj_types = ['services-tcp', 'services-udp', 'services-dce-rpc', 'services-rpc', 'services-other',
-                        'services-icmp', 'services-icmp6', 'services-sctp', 'services-gtp', 'Global']
+                        'services-icmp', 'services-icmp6', 'services-sctp', 'services-gtp']
 group_svc_obj_types = ['service-groups', 'application-site-categories', 'application-sites']
```

Comment:
line should be deleted I think

Reply:
sorry - which line do you suggest to delete? I just removed the "Global" table as it does not exist.

Reply:
you were fight - removed the redundant line